hello/hello_parallel_activity.py

```diff
@@ -0,0 +1,65 @@
+import asyncio
+from datetime import timedelta
+from typing import List
+
+from temporalio import activity, workflow
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+
+@activity.defn
+async def say_hello_activity(name: str) -> str:
+    return f"Hello, {name}!"
+
+
+@workflow.defn
+class SayHelloWorkflow:
+    @workflow.run
+    async def run(self) -> List[str]:
+        # Run 5 activities at the same time
+        results = await asyncio.gather(
```

Comment:
I'd mention `as_completed` as an alternative

Reply:
I think you'll be surprised by the amount of users that will ask you this question, leaving it up to you to decide